docs/commands/index.md

```diff
@@ -15,17 +16,17 @@ Redocly OpenAPI CLI currently supports the following commands:
 * [split](split.md)
 * [stats](stats.md)
 
-To define the behavior of the commands, you can use configuration files:
+## How configuration impacts commands
+OpenAPI CLI comes with two default configuration files:
 
-- `.redocly.yaml` - define the location of your root files, linting rules, and reference docs configuration information.
-- `.redocly.lint-ignore.yaml` - ignore specific lint messages.
+`redocly.yaml`
```

Comment:
issue: the configuration file is called `.redocly.yaml`. This might change relatively soon, but for now, removing the dot from the filename may cause problems.